readme/Installation.scalatex

```diff
@@ -81,7 +81,7 @@
       ./scalafix --help
 
     @p
```

Comment:
We can remove this paragraph altogether, the --scalahost-nsc-plugin-path flag is no longer necessary thanks to the fact that the scalameta semantic api no longer depends on scala-compiler or scalahost-nsc.